dashboard/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  Select as SelectUI,
+  SelectItem,
+  SelectContent,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+
+const Select = (props: React.ComponentProps<typeof SelectUI>): JSX.Element => {
+  const { children, ...propsWithoutChildren } = props;
+
+  return (
+    <SelectUI {...propsWithoutChildren}>
+      <SelectTrigger className="w-auto rounded-full border-2 border-dimGray px-6 py-4 text-base font-medium text-dimGray">
+        <SelectValue placeholder="Theme" />
```

Comment:
should we add a better placeholder and make it use i18n?

Reply:
actually this shouldn't be here, i was using it for testing purposes. Nice catch

Reply:
can we remove it?